examples/android/CHIPTool/app/src/main/java/com/google/chip/chiptool/clusterclient/MultiAdminClientFragment.kt

```diff
@@ -8,10 +8,14 @@ import android.view.ViewGroup
 import androidx.fragment.app.Fragment
 import androidx.lifecycle.lifecycleScope
 import chip.devicecontroller.ChipDeviceController
-import chip.devicecontroller.ClusterIDMapping
+import chip.devicecontroller.ClusterIDMapping.*
```

Comment:
could you import the specific things in clusterIdMapping instead of *? see https://developer.android.com/kotlin/style-guide, thanks

Reply:
Modified.